opentech/apply/users/views.py

```diff
@@ -71,6 +71,7 @@ def get(self, request, *args, **kwargs):
 
         if self.valid(user, kwargs.get('token')):
             user.is_active = True
+            user.set_unusable_password()
```

Comment:
The alternative is to not activate on this redirect view and only set them as active when they have set a password on `create_password` view.

Reply:
set_unusable_password() does not allow password reset to work, I could have sworn it worked when I first tested it but I must have done something wrong.